js/README.md

```diff
@@ -158,13 +184,20 @@ Index,   origin_city
 */
 ```
 
+# Tutorials and examples
+
+* [JavaScript Introduction to Arrow][6]
+* [Manipulating flat arrays arrow-style][7]
+* [/js/test/unit](https://github.com/apache/arrow/tree/master/js/test/unit) - Unit tests for Table and Vector
+
 # Getting involved
 
 See [develop.md](https://github.com/apache/arrow/blob/master/develop.md)
 
 Even if you do not plan to contribute to Apache Arrow itself or Arrow
 integrations in other projects, we'd be happy to have you involved:
 
+* [Join the Slack channel][5]: Channels `#general` and `#javascript`
```

Comment:
See recent mailing list thread about the Slack channel -- it's likely going to be closed soon